client/src/components/DecisionButtons.vue

```diff
@@ -0,0 +1,153 @@
+          code: 'U',
+          color: 'green darken-3 white--text',
+        },
+      ];
+      if (this.myCurrentProjectRoles.includes('tier_2_reviewer')) {
+        myOptions.push({
+          label: 'Usable-Extra',
+          code: 'UE',
+          color: 'grey darken-3 white--text',
+        });
+        myOptions.push({
+          label: 'Unusable',
+          code: 'UN',
+          color: 'red darken-3 white--text',
+        });
+      } else if (this.myCurrentProjectRoles.includes('tier_1_reviewer') || this.myCurrentProjectRoles.includes('superuser')) {
+        myOptions.push({
+          label: 'Questionable',
+          code: 'Q?',
+          color: 'grey darken-3 white--text',
+        });
+      }
+      return myOptions;
+    },
+  },
+  watch: {
+    newComment() {
+      this.warnDecision = false;
+    },
+  },
+  methods: {
+    handleCommentChange(value) {
+      this.newComment = value;
+    },
+    async handleCommentSave(decision) {
+      if (this.newComment.trim().length > 0 || decision === 'U') {
+        try {
+          const { addScanDecision } = store.commit;
+          const savedObj = await djangoRest.setDecision(
+            this.currentViewData.scanId,
+            decision,
+            this.newComment,
+          );
+          addScanDecision({
+            currentScan: this.currentViewData.scanId,
+            newDecision: savedObj,
+          });
+          this.$emit('handleKeyPress', 'next');
+          this.$snackbar({
+            text: 'Saved decision successfully.',
+            timeout: 6000,
+          });
+          this.warnDecision = false;
+          this.newComment = '';
+        } catch (err) {
+          console.log(err);
+          this.$snackbar({
+            text: `Save failed: ${err.response.data.detail || 'Server error'}`,
+            timeout: 6000,
+          });
+        }
+      } else {
+        this.warnDecision = true;
+      }
+    },
+  },
+};
+</script>
+
+<template>
+  <div style="pa-0 ma-0">
+    <v-row
+      v-if="experimentIsEditable"
+    >
+      <v-col
+        cols="12"
+        class="pb-0 mb-0"
+      >
+        <v-textarea
+          @input="handleCommentChange"
+          :counter="!warnDecision"
+          :hide-details="warnDecision"
+          v-model="newComment"
+          filled
+          no-resize
+          height="60px"
+          name="input-comment"
+          label="Evaluation Comment"
+          placeholder="Write a comment about the scan and submit a decision"
+        />
+      </v-col>
+    </v-row>
+    <v-row
+      v-if="warnDecision"
+      no-gutters
+    >
+      <v-col
+        cols="12"
+        class="red--text"
+        style="text-align: center"
+      >
+        Decisions other than "usable" must have a comment.
+      </v-col>
+    </v-row>
+    <v-row
+      v-if="experimentIsEditable"
+      no-gutters
+    >
+      <v-col
+        v-for="option in options"
+        :key="option.code"
+        :cols="12 / options.length"
```

Comment:
Even fully maximized, this is a little too cramped for 3 buttons:
![image](https://user-images.githubusercontent.com/577946/142659708-88e8348a-0f3d-4fe1-b7f1-dd64718a3a3f.png)
Perhaps there's some `flex` setting to make them collectively fill the available width? I'm not sure what would look best, especially on a smaller screen,

Reply:
Good catch!